docroot/modules/custom/dcc_gtd_registration/src/Form/GlobalTrainingRegistrationForm.php

```diff
@@ -0,0 +1,482 @@
+      '#title' => $this->t('Industry experience'),
+      '#type' => 'number',
+      '#min' => 0,
+      '#max' => 99,
+      '#field_suffix' => 'years',
+      '#default_value' => isset($industry_experience) ? $form_state->get("industry_experience") :NULL ,
+    );
+  }
+
+  /**
+   * Creates the form elements for the fourth step.
+   *
+   * @param array $form
+   *  The form elements.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *  The form state.
+   */
+  private function stepFourElements(array $form,FormStateInterface $form_state){
+    $form['container']['title'] = array(
+      '#title' => $this->t( "Training details"),
+      '#type' => 'item',
+    );
+    $attend_day = $form_state->get("attend_day");
+    $form['container']['attend_day'] = array(
+      '#title' => $this->t('On which days will you attend the training ?'),
+      '#description' => 'See the schedule!',
+      '#type' => 'checkboxes',
+      '#options' => array(
+        'Friday' => $this->t('Friday'),
+        'Saturday' => $this->t('Saturday')
+      ),
+      '#required' => TRUE,
+      '#default_value' => isset($attend_day) ? $form_state->get("attend_day") : NULL,
+    );
+    $options = array(0 => t('No'), 1 => t('Yes'));
+    $laptop=$form_state->get("laptop");
+    $form['container']['laptop'] = array(
+      '#title' =>  $this->t('Will you carry a laptop ?'),
+      '#description' => 'To know how many working stations will we need we should know if will you come with your own laptop. In case you will, please pre-configure your environment to be able to install Drupal 8 on it(if you need help check Drupa.org). Note: You will need your laptop for development only on Saturday',
+      '#type' => 'radios',
+      '#options' => $options,
+      '#required' => TRUE,
+      '#default_value' => isset($laptop) ? $form_state->get("laptop") : NULL,
+    );
+    $options2 = array(
+      'en' => t('English'),
+      'ro' => t('Romanian'),
+      'nr' => t('Not relevant')
+    );
+    $language=$form_state->get("language");
+    $form['container']['language'] = array(
+      '#title' =>  $this->t('Preferred langauge'),
+      '#description' => 'Please complete with your preferred language for presentations',
+      '#required' => TRUE,
+      '#type' => 'radios',
+      '#options' => $options2,
+      '#default_value' => isset($language) ? $form_state->get("language") : NULL,
+    );
+    $key_expectations=$form_state->get("key_expectations");
+    $form['container']['key_expectations'] = array(
+      '#title' => $this->t( 'Key expectations'),
+      '#type' => 'textarea',
+      '#description' => 'Please tell us about your expectations for the Global Training. Why are you enrolling?',
+      '#default_value' => isset($key_expectations) ? $form_state->get("key_expectations") : NULL,
+    );
+  }
+
+  /**
+   *Ajax callback method used for replacing the container with the form elements stored in $form['container'].
```

Comment:
I think the comment is too long, longer than 80 chars. Also, please add an empty space before Ajax.